tests/test_PenaltyManager.py

```diff
@@ -295,22 +296,23 @@ def test_does_not_update_penalties_before_sufficient_registrations(ok_small):
     assert_equal(pm.cost_evaluator().dist_penalty(1, 0), 2)
 
 
+@pytest.mark.filterwarnings("ignore::pyvrp.exceptions.PenaltyBoundWarning")
```

Comment:
This test warns but that isn't the core of the test, so I suppress the warning (else it shows up in the test summary and that's not really desired). The warning is tested in the new test below.